tests/recipes/test_freetype.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+from tests.recipes.recipe_lib_test import BaseTestForMakeRecipe
+
+
+class TestFreetypeRecipe(BaseTestForMakeRecipe, unittest.TestCase):
```

Comment:
Why do we need to inherit from `TestCase` suddenly?

Reply:
We actually need it because nor `RecipeCtx` nor `BaseTestForMakeRecipe` inherits from `unittest.TestCase`

Reply:
`<nitpick-troll>`
What is `TestCase` providing that you need in this case? Would tests fail or not run if we drop it? :trollface:
We don't even use setup/teardown here. Yes I know it's hard to say `unittest.TestCase` goodbye :joy: 
Honestly, feel free to not address that comment, I'm just challenging your habits :wink: 
`</nitpick-troll>`

Reply:
Jajajaja...No man, we need this or the tests won't run...we are using `setUp/teardown` for all library recipes we introduced/modified, since all of them inherits from `RecipeCtx`...or directly or via `BaseTestForMakeRecipe/BaseTestForCmakeRecipe`

...this fuc*** `Python Inheritance` makes the things hard to track sometimes...:laughing:

Reply:
Damn it, you got the point :joy: 
But yes definitely OOP sometimes :sweat_smile: 
Thanks man